straxen/contexts.py

```diff
@@ -218,19 +219,98 @@ def xenonnt_led(**kwargs):
     return st
 
 
-def xenonnt_simulation(output_folder='./strax_data'):
+def xenonnt_simulation(output_folder='./strax_data',
+                       cmt_run_id='020280',
+                       cmt_version='v3',
+                       fax_config='fax_config_nt_design.json',
+                       cmt_option_overwrite=immutabledict(),
+                       overwrite_from_fax_file=False,
+                       _cmt_run_id_proc_only=None,
```

Comment:
Should it be processing only or simulation only here? My understanding was that if we set `cmt_run_id`, it is used for simulation (inside wfsim etc) and in processing (in peak creation etc). And now when we set divergent `_cmt_run_id_proc_only`, does it invert the logic from previous comment? Or is it just naming?

Reply:
I think it's the same either way. (After the meeting I realized that cmt_run_id is tracked, as I pass the exact options to wfsim e.g. `'gain_model_mc': ('cmt_run_id', '020280', 'to_pe_model', 'v4', True)`. So we need another cmt_id if the processing is using some other run config.